### PR TITLE
Minor bug fixes.

### PR DESCRIPTION
**Pull request by Unengine**
### Description

SSIA

### How to test

1. Check if issues in 'Related Links' are resolved.

### Related Links
https://app.asana.com/0/1133453747809944/1201052373468536
https://app.asana.com/0/1133453747809944/1201074248891268

### Required Reviewers

@Namyujeong @planetarium/9c-dev

### CODE REVIEW

**Review by hskim881028 — APPROVED** [review]

File: nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlot.cs [hunk 1]
```diff
@@ -255,16 +255,22 @@ private void Awake()
                     image.gameObject.SetActive(false);
                 }
 
-                if (equip.optionCountFromCombination > 0)
+                var hasOptions = equip.optionCountFromCombination > 0 ||
+                    equip.StatsMap.GetAdditionalStats(true).Count() + equip.Skills.Count > 0;
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlot.cs:259]: `equip.optionCountFromCombination` does not work?

Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlot.cs:259]: This line is for backward compatibility. (legacy equipments)

**Comment by Namyujeong** [comment]
@Unengine Please link the version project from next time. This time, I linked it.

**Review by Namyujeong — APPROVED** [review]
@Unengine It successfully works! From next time, if there is a UI change, please upload a screenshot together.

스크린샷 2021-09-29 오후 9 23 53
스크린샷 2021-09-29 오후 9 24 44

**Pull request merged**